samples/Common.c

```diff
@@ -175,6 +179,35 @@ STATUS masterMessageReceived(UINT64 customData, PReceivedSignalingMessage pRecei
     return retStatus;
 }
 
+STATUS getSelectedIceCandidatesInformation(PSampleStreamingSession pSampleStreamingSession)
```

Comment:
Should the API be named as printSelectedIceCandidateInfo?

Reply:
Done